layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue

```diff
@@ -86,7 +86,7 @@ export default {
       return this.$t('siteNavigation.addElementDrawer.title');
     },
     disabled() {
-      return !this.isValidForm || !this.pageTemplate || false;
+      return !this.isValidForm || this.pageTemplate === 'empty'  || false;
```

Comment:
may be delete the default value 'empty' instead, WDYT ?

Reply:
done